Announce problem for second account

**Describe the bug**
There are no problem when first account is configured and tested. When the second one is added no problem with setup, however the notify (announce) service not working only for the second one. Others stuff as "last called", "volume control", etc working fine for both accounts, just the announce not working for the second one. No relevants logs errors catched (debug log enabled), so seems the integration "think itself" that is working properly. This happens for since v.3.5.0

**To Reproduce**
Steps to reproduce the behavior:
1. Delete alexa media player for both accounts
2. Delete alexa media player from HACS
3. Reboot
4. Add alexa integration from HACS
5. Reboot
6. Add 1st account integration, Echos devices detected, testing and seems works fine
7. Add 2nd account integration, Echos devices detected, testing announce but not working. Announce still working for the first account
8. Other services like volume, last_called, etc works for both account
9. Tested changing the account sequence when I register the integration. No matters wich account is the second one. The issue seems to be happen only with the second account

**Expected behavior**
Announce works for two accounts

**Screenshots**
N/A

**System details**
 - Home-assistant (version): 2021.1.5
 - Hassio (Yes/No): Yes
 - alexa_media  (version from `const.py` or HA startup): v3.5.2
 - alexapy (version from `pip show alexapy` or HA startup):
 - Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). We will not debug login issues if unanswered: Yes

**Logs**
Not sure if it is relevant or if it is related with Alexa Media Player, but they starting to appear with v3.5.2, but maybe are related with other stuffs:

```
2021-01-24 07:20:01 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/discovery.py", line 183, in async_load_platform
assert hass_config, "You need to pass in the real hass config"
AssertionError: You need to pass in the real hass config
```

**Additional context**
Amazon domain: .es
Rolling back to v.3.4.1 and working fine

--- a/custom_components/alexa_media/notify.py
+++ b/custom_components/alexa_media/notify.py
@@ -224,7 +224,7 @@ class AlexaNotificationService(BaseNotificationService):
                     #     list(map(hide_serial, targets)),
                     #     entities,
                     # )
-                    if alexa.unique_id in targets and alexa.available:
+                    if alexa.device_serial_number in targets and alexa.available:
                         _LOGGER.debug(
                             ("%s: Announce by %s to " "targets: %s: %s"),
                             hide_email(account),

